#! /bin/sh /usr/share/dpatch/dpatch-run
## fix_dllmap.dpatch by Mirco Bauer <meebey@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Tweak dllmap to better reflect the actual filename to bind to

@DPATCH@
Index: webkit-sharp/sources/webkit-sharp.dll.config.in
===================================================================
--- webkit-sharp.orig/sources/webkit-sharp.dll.config.in	2011-07-15 10:26:53.638141412 +0100
+++ webkit-sharp/sources/webkit-sharp.dll.config.in	2011-07-15 10:26:59.761901874 +0100
@@ -1,3 +1,3 @@
 <configuration>
-  <dllmap dll="webkit-1.0" target="webkit-1.0@LIB_PREFIX@.2@LIB_SUFFIX@"/>
+  <dllmap dll="webkit-1.0" target="libwebkitgtk-1.0@LIB_PREFIX@.0@LIB_SUFFIX@"/>
 </configuration>
